service/src/persist.rs

```diff
@@ -0,0 +1,92 @@
+use tokio::runtime::Runtime;
+
+use crate::{error::CustomError, Factory, ResourceBuilder};
+use async_trait::async_trait;
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+
+use std::fs;
+use std::fs::File;
+use std::io::BufReader;
+use std::io::BufWriter;
+
+use bincode::{deserialize_from, serialize_into};
+
+use shuttle_common::project::ProjectName;
```

Comment:
```suggestion
use tokio::runtime::Runtime;
use crate::{error::CustomError, Factory, ResourceBuilder};
use async_trait::async_trait;
use serde::de::DeserializeOwned;
use serde::Serialize;
use std::fs::{self, File, BufReader, BufWriter};
use bincode::{deserialize_from, serialize_into};
use shuttle_common::project::ProjectName;
```

Just rerun `cargo fmt` on this